2021/buvanesh2022re/openreview/packages.tex

```diff
@@ -0,0 +1,71 @@
+% Import packages here
+%\usepackage[utf8]{inputenc} % allow utf-8 input
+%\usepackage{hyperref}
+
+\usepackage{graphicx,booktabs,array}
+% if you need to pass options to natbib, use, e.g.:
+%     \PassOptionsToPackage{numbers, compress}{natbib}
+% before loading neurips_2019
+
+% ready for submission
+% \usepackage{neurips_2019}
+
+% to compile a preprint version, e.g., for submission to arXiv, add add the
+% [preprint] option:
+%     \usepackage[preprint]{neurips_2019}
+
+% to compile a camera-ready version, add the [final] option, e.g.:
+\usepackage[dvipsnames]{xcolor}
```

Comment:
Please remove this package, which is the cause of the issue (please check the README) : 

> Please do not import xcolor in your packages.tex, as rescience.cls already imports this package

Reply:
@koustuvsinha removing this also doesn't solve the issue on my end. Do you have any other pointers on what the issue could be?

Reply:
@anirudhb11 added a comment : comment out imports of bibliography in appendix.tex